pkg/transfers/inbound/cleanup.go

```diff
@@ -54,3 +78,41 @@ func deleteFilesOnRemote(logger log.Logger, agent upload.Agent, localDir, suffix
 	}
 	return el
 }
+
+// deleteEmptyFiles deletes all empty files that are older than after (time.Duration)
+func deleteEmptyFiles(logger log.Logger, agent upload.Agent, localDir, suffix string, now time.Time, after time.Duration) error {
+	baseDir := filepath.Join(localDir, suffix)
+	infos, err := ioutil.ReadDir(baseDir)
+	if err != nil {
+		return fmt.Errorf("reading %s: %v", baseDir, err)
+	}
+
+	var el base.ErrorList
+	for i := range infos {
+		fileInfo := infos[i]
+		path := filepath.Join(suffix, filepath.Base(infos[i].Name()))
+		if !shouldDeleteEmptyFile(fileInfo, now, after) {
```

Comment:
Why are we using `infos[i]` and `fileInfo` here? Can we use `infos[i]` consistently.

Reply:
Good catch. Pushed.